components/CaseStatus/EditCaseStatusForm/EditCaseStatusForm.tsx

```diff
@@ -61,6 +75,52 @@ const EditCaseStatusForm: React.FC<{
     });
   }
 
+  if (caseStatusType == 'LAC' && (action == 'update' || action == 'end')) {
+    if (
+      currentCaseStatusStartDate &&
+      currentCaseStatusStartDate !== 'undefined'
+    ) {
+      const currentStartDateValidation = new Date(currentCaseStatusStartDate);
+
+      if (action == 'update') {
+        currentStartDateValidation.setDate(
+          currentStartDateValidation.getDate() + 1
+        );
+      }
+
+      form_fields.map((field: any) => {
+        if (field.id === 'startDate' || field.id === 'endDate') {
+          field.startDate = format(
+            new Date(currentStartDateValidation),
+            'yyyy-MM-dd'
+          );
+
+          field.default = format(
+            new Date(currentStartDateValidation),
+            'yyyy-MM-dd'
+          );
+        }
+      });
+    }
+  }
+
+  if (caseStatusType == 'LAC' && action == 'edit') {
```

Comment:
Rachel and I were wondering that if in the scenario **where we have a current case status & a past case status:**
if the validation date should be the past case status start date + 1
& also
If the date picker default date should be the current case status start date, as this may be clearer that we are editing this date value by doing this action.

```
if (caseStatusType == 'LAC' && action == 'edit') {
    if (pastCaseStatusStartDate && pastCaseStatusStartDate !== 'undefined' && currentCaseStatusStartDate? && currentCaseStatusStartDate !== 'undefined') {
      const defaultStartDate = new Date(currentCaseStatusStartDate);
      const pastStatusStartDate = new Date(pastCaseStatusStartDate);

      pastStatusStartDate.setDate(pastStatusStartDate.getDate() + 1);

      form_fields.map((field: any) => {
        if (field.id === 'startDate') {
          field.startDate = format(pastStatusStartDate, 'yyyy-MM-dd');

          field.default = format(defaultStartDate, 'yyyy-MM-dd');
        }
      });
    }
  }
```

 And then in the scenario **where we have a current case status only:**
 
```
const defaultStartDate = new Date(currentCaseStatusStartDate);
 field.default = format(defaultStartDate, 'yyyy-MM-dd');
```

Reply:
I don't think it should be the past case status start date +1, as you may want to edit only the two dropdowns while keeping the same date (so no +1 involved)

I'll update anyways the code to have as default the case status start date, sounds good to me :)